CHANGELOG.md

```diff
@@ -694,7 +696,7 @@ peripheral.register.write(|w| w.field().set());
 
 - Initial version of the `svd2rust` tool
 
-[Unreleased]: https://github.com/rust-embedded/svd2rust/compare/v0.23.1...HEAD
+[unreleased]: https://github.com/rust-embedded/svd2rust/compare/v0.23.1...HEAD
```

Comment:
Why you broke this link?

Reply:
OMG, I accidentally format the whole changelog. 😢 

Will fix it ASAP.

Reply:
Fixed. I forgot the formatting tool is still on.